src/Sub/Queue/Jobs/SqsSnsJob.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace PodPoint\AwsPubSub\Sub\Queue\Jobs;
+
+use Aws\Sqs\SqsClient;
+use Illuminate\Container\Container;
+use Illuminate\Events\CallQueuedListener;
+use Illuminate\Queue\Jobs\SqsJob;
+use Illuminate\Support\Arr;
+
+class SqsSnsJob extends SqsJob
+{
+    /**
+     * Create a new job instance.
+     *
+     * @param  \Illuminate\Container\Container  $container
+     * @param  \Aws\Sqs\SqsClient  $sqs
+     * @param  array  $job
+     * @param  string  $connectionName
+     * @param  string  $queue
+     * @return void
+     */
+    public function __construct(
+        Container $container,
+        SqsClient $sqs,
+        array $job,
+        $connectionName,
+        $queue
```

Comment:
ConnectionName and queue have no type here.

Reply:
We inherit from Laravel's one and Laravel's one doesn't have types either so we cannot add them otherwise it will be marked as "incompatible".

https://github.com/laravel/framework/blob/8.x/src/Illuminate/Queue/Jobs/SqsJob.php#L35